src/window.rs

```diff
@@ -1389,3 +1388,21 @@ impl Default for UserAttentionType {
         UserAttentionType::Informational
     }
 }
+
+/// A window level groups windows with respect to their z-position.
+///
+/// The relative ordering between windows in different window levels is fixed.
+/// The z-order of a window within the same window level may change dynamically on user interaction.
+///
+/// ## Platform-specific
+///
+/// - **iOS / Android / Web / Wayland:** Unsupported.
+#[derive(Debug, PartialEq, Eq, Clone, Copy)]
+pub enum WindowLevel {
```

Comment:
Could you impl `Default` for it and then use `Default::default()` instead of `Normal`?